src/core/qgsrulebasedlabeling.h

```diff
@@ -375,7 +375,7 @@ class CORE_EXPORT QgsRuleBasedLabeling : public QgsAbstractVectorLayerLabeling
     void toSld( QDomNode &parent, const QgsStringMap &props ) const override;
 
   protected:
-    Rule *mRootRule = nullptr;
+    std::unique_ptr<Rule> mRootRule;
```

Comment:
Can you check the blame here? I vaguely recall doing something similar and it being reverted later for some reason

Reply:
Nothing can be found.

The only change you did was initialize to nullptr at some point.

Reply:
Right, proceed away then :)